next/src/components/cards/EventCard.tsx

```diff
@@ -1,67 +1,83 @@
 import { Typography } from '@bratislava/component-library'
 import Image from 'next/image'
-import React, { HTMLAttributes } from 'react'
+import React, { useId } from 'react'
 
 import CardBase from '@/src/components/cards/CardBase'
-import CardContent from '@/src/components/cards/CardContent'
-import MLink from '@/src/components/common/MLink/MLink'
-import FormatEventDateRange from '@/src/components/formatting/FormatEventDateRange'
-import cn from '@/src/utils/cn'
+import { CardTitleLevel } from '@/src/components/cards/getCardTitleLevel'
+import Button from '@/src/components/common/Button/Button'
+import ImagePlaceholder from '@/src/components/common/Image/ImagePlaceholder'
+import { EventPageCardEntityFragment } from '@/src/services/graphql'
+import { formatDate } from '@/src/utils/formatDate'
+import { generateImageSizes } from '@/src/utils/generateImageSizes'
 import { getLinkProps } from '@/src/utils/getLinkProps'
 import { useTranslation } from '@/src/utils/useTranslation'
 
-type EventCardProps = {
-  title: string
-  linkHref: string
-  address?: string
-  dateFrom: string
-  dateTo: string | null
-  isLongTerm?: boolean
-  imageSrc: string
-  imageSizes?: string
-  className?: string
-} & HTMLAttributes<HTMLElement>
+type Props = {
+  eventPage: EventPageCardEntityFragment
```

Comment:
I find the name "event page" a bit confusing, it looks like it is a separate content type. I don't know what to do with it, but I wanted to point it out.